p2pserver/message/utils/msg_handler.go

```diff
@@ -642,3 +642,31 @@ func GetHeadersFromHash(startHash common.Uint256, stopHash common.Uint256) ([]*t
 
 	return headers, nil
 }
+
+// EmergencyRequestHandle forwards emergency request msg to emergency governance module to handle
+func EmergencyRequestHandle(data *msgTypes.MsgPayload, p2p p2p.P2P, pid *evtActor.PID, args ...interface{}) {
+	log.Debug("RX emergency request message", data.Addr, data.Id)
+
+	if pid != nil {
+		msg := data.Payload.(*msgTypes.EmergencyReqMsg)
+		input := &msgCommon.EmergencyMsg{
+			MsgType: msgCommon.EmergencyReq,
+			Content: &msg.Payload,
+		}
+		pid.Tell(input)
+	}
+}
+
+// EmergencyResponseHandle forwards emergency response to emergency governance module to handle
+func EmergencyResponseHandle(data *msgTypes.MsgPayload, p2p p2p.P2P, pid *evtActor.PID, args ...interface{}) {
+	log.Debug("RX emergency response message", data.Addr, data.Id)
+
+	if pid != nil {
+		msg := data.Payload.(*msgTypes.EmergencyRspMsg)
+		input := &msgCommon.EmergencyMsg{
+			MsgType: msgCommon.EmergencyRsp,
+			Content: &msg.Payload,
+		}
+		pid.Tell(input)
```

Comment:
verify the response?

Reply:
Same as above. It is handled in the emergency goverance module.